templates/config.edn

```diff
@@ -146,33 +176,38 @@
  ;;  :editor/right           ["ctrl+l" "right"]}
  :shortcuts {}
 
- ;; By default, pressing `Enter` in the document mode will create a new line.
- ;; Set this to `true` so that it's the same behaviour as the usual outliner mode.
+ ;; Configure the behavior of pressing Enter in document mode.
+ ;; if set to true, pressing Enter will create a new block.
+ ;; Default value: false
  :shortcut/doc-mode-enter-for-new-block? false
 
- ;; Block content larger than `block/content-max-length` will not be searchable
- ;; or editable for performance.
+ ;; Limit the searchable and editable block content length for performance
+ ;; Default value: 10000
  :block/content-max-length 10000
 
- ;; Whether to show command doc on hover
+ ;; Display command documentation on hover.
+ ;; Default value: true
  :ui/show-command-doc? true
 
- ;; Whether to show empty bullets for non-document mode (the default mode)
+ ;; Display empty bullet points.
  :ui/show-empty-bullets? false
```

Comment:
the default value is nil or false

https://github.com/logseq/logseq/blob/a5e31128a6366df002488203406684f78d80c7e3/src/main/frontend/components/block.cljs?plain=1#LL1770C1-L1774C16
```cljs
       (cond
         (and (or (mobile-util/native-platform?)
                  (:ui/show-empty-bullets? (state/get-config)))
              (not doc-mode?))
         bullet
```